cdmtaskservice/coroutine_manager.py

```diff
@@ -0,0 +1,59 @@
+"""
+A manager of coroutines.
```

Comment:
How do you plan to use CoroutineWrangler? As a context manager, like this:
`async with CoroutineWrangler() as manager:`? If that's the case, I think `__aexit__` should be overwritten with `await self.destroy()`. Probably also implement `__aenter__`

Reply:
In the same was as the other classes in `app_state`. No context manager